.prettierignore

```diff
@@ -1,2 +1,4 @@
 *.css
-*.scss
\ No newline at end of file
+*.scss
+*.sass
+*.json
```

Comment:
should JSON config files also be formatted with prettier? like the 4 spaces for tabs

Reply:
i dont think stylelint formats json so i think yeah it can be